### PR TITLE
Fix #262 Updating CI config

### LINKED ISSUE #262: Update CI setup configuration

### PR DESCRIPTION
**Pull request by hferentschik**
The important change is to remove the explicit specification of the bundler version. This is already applied on the CI environment. 

The other things are minor improvement and documentation changes.

### CODE REVIEW

**Comment by hferentschik** [comment]
please test

**Comment by hferentschik** [comment]
retest this please

File: .ci/ansible/roles/centos/tasks/main.yml [hunk 1]
```diff
@@ -19,8 +19,9 @@
       - gcc
       - kernel-devel
 
-- name: Install VirtualBox
-  yum: name=http://download.virtualbox.org/virtualbox/rpm/rhel/7/x86_64/VirtualBox-5.0-5.0.8_103449_el7-1.x86_64.rpm state=present
+# TODO enable when we tst against VirtualBox as well
```
Comment (brgnepal) [inline .ci/ansible/roles/centos/tasks/main.yml]: typo in test.

**Comment by brgnepal** [comment]
Code LGTM. Since tests are passing we are good to go.

**Comment by brgnepal** [comment]
Fix #262

**Comment by hferentschik** [comment]
@budhrg didn't you want to merge this one?

**Comment by brgnepal** [comment]
Merged

**Pull request closed without merging**